packages/big-design/src/components/Button/spec.tsx

```diff
@@ -189,19 +189,6 @@ describe('isLoading', () => {
     expect(screen.getByRole('button')).toMatchSnapshot();
   });
 
-  test('prevents on click', () => {
-    const mockClickHandler = jest.fn();
-    render(
-      <Button isLoading={true} onClick={mockClickHandler}>
-        Button
-      </Button>,
-    );
-
-    userEvent.click(screen.getByRole('button'), undefined, { skipPointerEventsCheck: true });
-
-    expect(mockClickHandler).not.toHaveBeenCalled();
-  });
-
```

Comment:
Removed because `skipPointerEventsCheck` is no longer a thing (?? not sure about this), so it would error out.

Reply:
Could we just get rid of the `{ skipPointerEventsCheck: true }` object instead of the test?